composer.json

```diff
@@ -88,7 +88,7 @@
         "matthiasnoback/symfony-dependency-injection-test": "^1.0",
         "mikey179/vfsStream": "^1.6",
         "pamil/prophecy-common": "^0.1",
-        "phpspec/phpspec": "^4.0",
+        "phpspec/phpspec": "^4.0@dev",
```

Comment:
Same change applies for every Sylius package using phpspec (packages build on PHP 7.2 would fail without this one).

Reply:
Alright. Need to see if this change is required at all before finalising PR.